migrate/migrator.go

```diff
@@ -362,7 +362,9 @@ func (m *Migrator) MarkUnapplied(ctx context.Context, migration *Migration) erro
 }
 
 func (m *Migrator) TruncateTable(ctx context.Context) error {
-	_, err := m.db.NewTruncateTable().TableExpr(m.table).Exec(ctx)
+	_, err := m.db.NewTruncateTable().
+		ModelTableExpr(m.table).
+		Exec(ctx)
```

Comment:
This does not look right, because there is no `Model` call. Could you revert it?

Reply:
@vmihailenco 
Thanks for the reply.

Referring to other implementations, I decided that it would be better to call a Model method than to revert.
I would appreciate it if you could check the following commit again. https://github.com/uptrace/bun/pull/969/commits/27193c377d854c591f07b640349d8de7f9bf5bb0